packages/thumbprint-react/components/BlockList/index.module.scss

```diff
@@ -0,0 +1,35 @@
+@import '~@thumbtack/thumbprint-tokens/dist/scss/_index';
+
+.list {
+    border-top: 1px solid $tp-color__gray-300;
+}
+
+.listBorderGroup {
+    border: 1px solid $tp-color__gray-300;
+    border-radius: 4px;
+}
+
+.listBorderBottom {
+    border-bottom: 1px solid $tp-color__gray-300;
+}
+
+.listFlush {
+    .item {
+        padding-left: 0;
+        padding-right: 0;
+    }
+}
+
+.item {
+    padding: $tp-space__2 $tp-space__3;
+
+    &:not(:first-child) {
+        border-top: 1px solid $tp-color__gray-300;
+    }
+}
+
+.itemLink {
+    padding: $tp-space__2 $tp-space__3;
```

Comment:
Should we bump up the specificity here to prevent people from overriding these values? Similar to this approach: https://github.com/thumbtack/thumbprint/blob/519ff414afd76aff9cb98d7bddf1487c62386f0f/packages/thumbprint-react/components/Type/index.module.scss#L4-L16

Reply:
I thought about that, couldn't decide. And there is at least one case where the padding was overridden. No strong feelings.

Reply:
Probably worth doing so that we can more confidently refactor in the future. If it needs to be overwritten and there's a good use case, we could add that as a prop in the future.

Reply:
That would apply to every class in the file, yes?

Reply:
It'd be specifically for `BlockListItemLink` since that's the only one where devs can pass in custom classes.

Reply:
Oh, of course. Thanks.

Reply:
The `!important` part is needed too so that Atomic can't override it.

Reply:
Added